server/opts.go

```diff
@@ -479,6 +480,10 @@ func (o *Options) ProcessConfigFile(configFile string) error {
 		o.processConfigFileLine(k, v, &errors, &warnings)
 	}
 
+	if len(errors) == 0 {
```

Comment:
I tend to prefer doing these kind of checks in `validateOptions()`. The reason is that it will cover both config file parsing but also misconfiguration for users (or tests) embedding server and configuring through options.

Reply:
Much better place, wil add. It appears that validateOptions() calls out to functions that are located where the option applies. So, I will move all the checks in a function in auth.go

Reply:
@kozlovic , I moved the checks. Better place an all... But it appears to me that this function is not called during reload. So this new check as well as the ones that were there before won't be called. I'll fix that in a separate PR.